### PR TITLE
Update Operator Lifecycle Manager Exercise for 4.4

### PR DESCRIPTION
**Pull request by madorn**
This PR updates the Operator Lifecycle Manager Exercise to run on OCP 4.4 environments.

### CODE REVIEW

File: operatorframework/operator-lifecycle-manager-42/index.json [hunk 1]
```diff
@@ -8,20 +8,29 @@
...
     },{
-			"title": "Setting up the UI",
-			"text": "step2.md"
-		},{
-			"title": "Creating the Subscription",
-			"text": "step3.md"
+      "title": "CatalogSources",
+      "text": "step2.md"
     },{
-			"title": "Approving the InstallPlan",
```
Comment (btannous) [inline operatorframework/operator-lifecycle-manager-42/index.json:20]: just cleaning up the tab/space mixing in this.

File: operatorframework/operator-lifecycle-manager-42/step2 copy.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+Observe the CatalogSources that ship with OLM and OpenShift 4:
```
Comment (btannous) [inline operatorframework/operator-lifecycle-manager-42/step2 copy.md:1]: @madorn why is this file here?

Comment (btannous) [reply operatorframework/operator-lifecycle-manager-42/step2 copy.md:1]: not sure if it's needed or not

Comment (btannous) [reply operatorframework/operator-lifecycle-manager-42/step2 copy.md:1]: that being said, there is some html tags in here.

File: operatorframework/operator-lifecycle-manager-42/step2.md [hunk 3]
```diff
@@ -1,24 +1,34 @@
+Observe the CatalogSources that ship with OLM and OpenShift 4:
 
-To access the Operator Lifecycle Manager via the UI, we will run a local copy of the UI found in the container image `quay.io/openshift/origin-console:latest`.
+```
+oc get catalogsources -n openshift-marketplace
+```{{execute}}
+<br>
```
Comment (btannous) [inline operatorframework/operator-lifecycle-manager-42/step2.md:6]: html tag? we can get rid of this and add a double return instead

File: operatorframework/operator-lifecycle-manager-42/step3 copy.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+You can access OLM via the OpenShift console by clicking the [Console](https://console-openshift-console-[[HOST_SUBDOMAIN]]-443-[[KATACODA_HOST]].environments.katacoda.com) tab to open the dashboard.
+
+You will then able able to login with admin permissions:
+
+* **Username:** ``admin``{{copy}}
+* **Password:** ``admin``{{copy}}
+
+The OLM panel appears in the **Operators** pane:
+
+![OLM on the OpenShift Console](https://raw.githubusercontent.com/madorn/learn-katacoda/master/operatorframework/operator-lifecycle-manager/assets/images/olm-console.png)
```
Comment (btannous) [inline operatorframework/operator-lifecycle-manager-42/step3 copy.md:10]: not sure how other images in here are linked, but it would be better to move these to a corresponding asset directory: here https://github.com/openshift-labs/learn-katacoda/tree/master/assets that way they get served by katacoda

link it this way: https://github.com/openshift-labs/learn-katacoda/blame/master/developing-on-openshift/helm/01-cli.md#L15

File: operatorframework/operator-lifecycle-manager-42/step3.md [hunk 5]
```diff
@@ -1,35 +1,10 @@
...
-```{{execute}}
-<br>
-Verify the Subscription and InstallPlan have been created.
+The OLM panel appears in the **Operators** pane:
 
-```
-oc get subscription
-oc get installplan
-```{{execute}}
+![OLM on the OpenShift Console](https://raw.githubusercontent.com/madorn/learn-katacoda/master/operatorframework/operator-lifecycle-manager/assets/images/olm-console.png)
```
Comment (btannous) [inline operatorframework/operator-lifecycle-manager-42/step3.md:10]: see previous comment about images.

File: operatorframework/operator-lifecycle-manager-42/step4 copy.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
+Let's begin my creating a new project called `myproject`.
+
+```
+oc new-project myproject
+```{{execute}}
+<br>
```
Comment (btannous) [inline operatorframework/operator-lifecycle-manager-42/step4 copy.md:6]: html tag, there are a few of them in this file

Comment (btannous) [reply operatorframework/operator-lifecycle-manager-42/step4 copy.md:6]: also, do we need the "copy" file anyways?

File: operatorframework/operator-lifecycle-manager-42/step4.md [hunk 7]
```diff
@@ -1,16 +1,68 @@
-Modify the InstallPlan and set `approved` to `true`.
+Let's begin my creating a new project called `myproject`.
 
 ```
-oc edit InstallPlan
+oc new-project myproject
 ```{{execute}}
 <br>
```
Comment (btannous) [inline operatorframework/operator-lifecycle-manager-42/step4.md:6]: html tags

File: operatorframework/operator-lifecycle-manager-42/step5.md [hunk 8]
```diff
@@ -1,27 +1,53 @@
...
+
+```
+oc get rolebindings | grep argocd
+```{{execute}}
 
-Confirm the cluster has been created:
+```
+oc get deployments
+```{{execute}}
+<br>
```
Comment (btannous) [inline operatorframework/operator-lifecycle-manager-42/step5.md:45]: html tags

File: operatorframework/operator-lifecycle-manager-42/step6 copy.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+    version: openshift-connector
+  rbac:
+    policy: |
+      g, system:cluster-admins, role:admin
+  server:
+    route:
+      enabled: true
+EOF
+```{{execute}}
+<br>
```
Comment (btannous) [inline operatorframework/operator-lifecycle-manager-42/step6 copy.md:23]: html tags and copy file?

File: operatorframework/operator-lifecycle-manager-42/step6.md [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+    version: openshift-connector
+  rbac:
+    policy: |
+      g, system:cluster-admins, role:admin
+  server:
+    route:
+      enabled: true
+EOF
+```{{execute}}
+<br>
```
Comment (btannous) [inline operatorframework/operator-lifecycle-manager-42/step6.md:23]: html tags

File: operatorframework/operator-lifecycle-manager/index.json [hunk 11]
```diff
@@ -8,20 +8,29 @@
   "details": {
```
Comment (btannous) [inline operatorframework/operator-lifecycle-manager/index.json:8]: indention here have many tab/spaces mixed

File: operatorframework/operator-lifecycle-manager-42/step2 copy.md (hunk 2 above)
Comment (madorn) [reply operatorframework/operator-lifecycle-manager-42/step2 copy.md]: inclusion of copy's was a mistake - going to remove entire 42 since its not necessary.

**Comment by madorn** [comment]
@btannous thanks for reviewing, fixed requested changes, removed operator-lifecycle-manager-42 dir.
